github/ci/services/prometheus-stack/manifests/service-monitors/production-control-plane/prow-exporter.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: prow-exporter
+  labels:
+    group: kubevirtci
+spec:
+  selector:
+    matchLabels:
+      app: prow-exporter
```

Comment:
Would it make sense to simply create on `ServiceMonitor` and just add labels to the appropriate services?

Reply:
Sorry, I don't follow, we are creating here a `ServiceMonitor` and a service with the matching labels, is that what you mentioned?

Reply:
I just meant that this seems to apply the pattern of having one service monitor per service. I just meant that you may want to create a `prow` ServiceMonitor which matches a more generic label which is not on the app base.

Reply:
Aha, sure makes sense to have a common ServiceMonitor for all of our services, I've pushed changes for this, the only requirements for the services to be handled by it are the `monitoring: "true"` label and having a port named `metrics` PTAL.